README.md

```diff
@@ -1,4 +1,41 @@
 # The Roux Reader [![Build Status](https://travis-ci.com/rouxers/book.svg?branch=master)](https://travis-ci.com/rouxers/book)
-Answers to all your Roux questions in one place
+Answers to all your Roux questions in one place.
+
+[Website](https://book.rouxers.com)
+
+====
+
+## Introduction
+Cubing resources are easy to procure, but they often lack the proper **annotations** to explain the context -- When should I learn this? How much faster will I be? Is X the only way to Y? What sort of progression should I expect? Is this the best quality resource out there? As a result, cubers might get confused about what to learn next, or spent time learning stuff way too advanced for their level.
+
+That's the motivation behind the Roux Reader. It hopes to fill this gap for Roux method speedsolvers -- it is the missing **user manual** that explains **how to pick the top quality resources that are best suited to your skill level**, and how to set expectations when learning them.
+
+
+## How to Read the Reader?
+
+First off, this reader is written for Roux users of all levels, as long as you know beginner Roux. (If not, check out this awesome beginner tutorial [guide](https://tutorial.rouxers.com) here).
+
+There isn't a single recommended way of reading it, but here are some useful scenarios:
+
+- If you already have a topic in mind (e.g. FB turning, EOLR progression), go find the related sections, and read the explanations to help you select the best resources.
+- If you are just randomly looking for things to learn, skim through the book for whatever interests you, and read up the section overview to discover exciting skills, techniques or concepts to work on.
+
+- If you have a particular question in mind, it might have been asked by many before. That's why we have created the [FAQ chapter](book.rouxers.com/faqs.html). Try your luck there!
+
+## Organization
+
+At the top level, the reader is organized by the four steps in Roux -- FB, SB, CMLL & LSE, plus a few chapters for generic topics that don't belong to a step. In Blockbuilding, we go over the three fundamental aspects of speedcubing -- **solutions, fingertricks, and lookahead**. For each of these subsections, we first provide an **overview** of what a fast cuber should be ideally doing, and talk about the **progression** -- what do learn at different skill levels. This overview is followed by a curated list of **resources** with commentaries. In CMLL and LSE, this structure is generally followed, with the additional emphasis on sub-step variants and their algorithms. Lastly, while OH and 2H techniques can differ sometimes, they are usually discussed within the same subsections to avoid fragmenting.
```

Comment:
to avoid fragmenting -> to avoid fragmentation?

Reply:
yes